src/main/content/_config.yml

```diff
@@ -11,6 +11,10 @@ slack_url: https://ibm-cloud-tech.slack.com/messages/CJZCYTD0Q
 slack_invite_url: https://slack-invite-ibm-cloud-tech.mybluemix.net/
 twitter_username: KabaneroIO
 github_username: kabanero-io
+docs_url: /docs
+title_logo: /img/Kabanero_Logo_Hero.png
+header_logo: /img/Kabanero_Logo_White_Text.png
```

Comment:
I would suggest creating a new section in here called `site_custom` and have a page title for each custom thing in each page (like we do for translation) there will be a lot more possible in the future so it would be nice to organize them by page